command/build.py

```diff
@@ -1,8 +1,39 @@
+import yaml as pyYaml
+import command.identify as identify
+
 class BuildCmd():
     name = 'build'
-
-    def __init__(self):	
+    
+    def __init__(self):
         pass
+
+    def _get_empty_bundle(self):
+        operatorArtifactsTemplate = dict(
+            data = dict(
+                customResourceDefinitions = [],
+                clusterServiceVersions = [],
+                packages = [],
+            )
+        )
+        return operatorArtifactsTemplate
     
-    def build(self):
-        print("Hello, world. The build feature is currently under construction.")
+    def _get_field_entry(self, yamlContent):
+        yaml_type = identify.get_operator_artifact_type(yamlContent)
+        if yaml_type == "ClusterServiceVersion" or yaml_type == "CustomResourceDefinition" or yaml_type == "Package":
+            return yaml_type[0:1].lower() + yaml_type[1:] + 's'
+
+    def _updateArtifact(self, operatorArtifact, yamlContent):
```

Comment:
I assume the underscore means this is a private method, in these cases do we not add method descriptions?

Reply:
@awgreene yes that is the case.